mod-di-converter-storage-server/src/main/resources/templates/db_scripts/data-migration/add_donor_organizations_ids_field_mapping.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE ${myuniversity}_${mymodule}.mapping_profiles
+SET jsonb = jsonb_insert(jsonb, '{mappingDetails, mappingFields, 44}', '{"name": "donorOrganizationIds", "enabled": "true", "required": false, "path": "order.poLine.donorOrganizationIds[]", "value": "", "subfields": []}')
```

Comment:
@RuslanLavrov  Do we need to add a comma (,) here? I mean between json objects in the list of mappingFields?

Reply:
@RuslanLavrov  I seems like it will be added by jsonb_insert.